nextflow_schema.json

```diff
@@ -38,6 +38,118 @@
                 }
             }
         },
+        "database": {
+            "title": "Databases",
+            "type": "object",
+            "description": "Select databases to be run on all samples.",
+            "fa_icon": "fas fa-terminal",
+            "properties": {
+                "pointfinder_database": {
+                    "enum": [
+                        "Automatic Selection",
+                        "Enterococcus faecium",
+                        "Enterococcus faecalis",
+                        "Helicobacter pylori",
+                        "Salmonella",
+                        "Campylobacter",
+                        "Escherichia coli"
+                    ],
+                    "description": "Select a single Pointfinder database to use on all samples (overriding metadata option). Validated Organisms: Enterococcus faecium, Enterococcus faecalis, Helicobacter pylori, Salmonella, Campylobacter, Escherichia coli"
+                },
+                "plasmidfinder_database": {
+                    "enum": ["All", "gram_positive", "enterobacteriales"],
+                    "description": "The database type to use for plasmidfinder {gram_positive, enterobacteriales}. Defaults to using all available database types to search for plasmids. [All]."
+                },
+                "mlst_scheme": {
+                    "type": "string",
+                    "description": "Specify scheme name, visit https://github.com/tseemann/mlst/tree/master/db/pubmlst for supported scheme genera available. [None]",
+                    "default": "Automatic"
+                }
+            }
+        },
+        "additional_settings": {
+            "title": "Additional Settings",
+            "type": "object",
+            "description": "For advanced changes to staramr",
+            "properties": {
+                "genome_size_lower_bound": {
+                    "type": "integer",
+                    "description": "The lower bound for our genome size for the quality metrics [Default 4000000]",
+                    "default": 4000000,
```

Comment:
As above with the default and Campys length being ~1,600,000. I wonder if this should be more like 1,000,000?